src/cli/subcommand/install.rs

```diff
@@ -86,18 +87,15 @@ impl CommandExecute for Install {
                 match existing_receipt {
                     Some(existing_receipt) => {
                         if existing_receipt.planner.typetag_name() != chosen_planner.typetag_name() {
-                            eprintln!("{}", format!("Found existing plan in `{RECEIPT_LOCATION}` which used a different planner, try uninstalling the existing install").red());
+                            eprintln!("{}", format!("Found existing plan in `{RECEIPT_LOCATION}` which used a different planner, try uninstalling the existing install with `/nix/nix-installer uninstall`").red());
```

Comment:
Maybe this should go in a future PR, but we should probably also check that the binary exists too... Otherwise, suggest getting the matching version from the receipt.